src/stk/_internal/topology_graphs/cage/one_plus_one.py

```diff
@@ -171,10 +174,96 @@ class OnePlusOne(Cage):
 
     """
 
+    def __init__(
+        self,
+        building_blocks: Iterable[BuildingBlock]
+        | dict[BuildingBlock, tuple[int, ...]],
+        vertex_alignments: dict[int, int] | None = None,
+        reaction_factory: ReactionFactory = GenericReactionFactory(),
+        num_processes: int = 1,
+        optimizer: Optimizer = NullOptimizer(),
+    ) -> None:
+        """
+        The `vertex_positions` argument is not available for this class
```

Comment:
I would reomve this from a docstring -- if you wanna keep the coment maybe add it as a code  comment or under a Notes section